src/main/java/Duke.java

```diff
@@ -1,10 +1,221 @@
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+import java.util.ArrayList;
+import java.io.IOException;
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-    }
-}
+    public static void main(String[] args) throws IOException {
+        Scanner in = new Scanner(System.in);
+        int tasksSize = 100;
```

Comment:
Constant?
In fact, since you are using arraylist, this may not be relevant, is it?